configs/pulpcore/3.15.yaml

```diff
@@ -0,0 +1,130 @@
+---
+:project: pulpcore
+:github_org: pulp
+:nightly: false
+:repos: []
+:releases:
+  :3.15:
+    :redmine_version_id: -1
+:mash_scripts:
+  - pulpcore-3.15-el7
+  - pulpcore-3.15-el8
+:strict_keys: true
+:gpg_key: 6D05E64C
+:tags:
+  - name: pulpcore-3.15-el7
+    based_off: pulpcore-3.14-el7
+    helper_tags:
+      pulpcore-3.15-el7-override: null
+    arches:
+      - x86_64
+    build_target: pulpcore-3.15-el7-build
+    inherits:
+      pulpcore-3.15-el7-build:
+        pulpcore-3.15-el7-override: 0
+      pulpcore-3.15-el7-override:
+        pulpcore-3.15-el7: 0
+    external_repos:
+      - centos7-os
+      - centos7-updates
+      - centos-sclo-rh-rhel-7
+      - epel-7
+    build_groups:
+      build:
+        - bash
+        - bzip2
+        - coreutils
+        - cpio
+        - diffutils
+        - findutils
+        - gawk
+        - gcc
+        - gcc-c++
+        - grep
+        - gzip
+        - info
+        - make
+        - patch
+        - python3-rpm-macros
+        - redhat-release
+        - redhat-rpm-config
+        - rpm-build
+        - rh-python38-scldevel
```

Comment:
I think this should be `rh-python38-build`

Reply:
We're going to create a `tfm-pulpcore` SCL, and for Foreman we include `rh-ruby27-scldevel` and `tfm-build`, so I am going to assume we would want `rh-python38-scldevel` and `tfm-pulpcore-build`.

Given the later doesn't exist (yet), I am going to merge and run this as-is, then build the SCL package, and then add it here.